tools/attest/attest.go

```diff
@@ -130,8 +131,9 @@ func parseBytes(name string, in io.Reader, inform string, byteSize int) ([]byte,
 	}
 }
 func main() {
-	logger.Init("", *verbose, false, os.Stderr)
 	flag.Parse()
+	logger.Init("", *verbose, false, os.Stderr)
```

Comment:
verbose argument is used to write info and warning logs to the log file i.e. STDERR in this case.  It uses default logger created using Init call.
Whereas verbosity level defines the detail level upto which we want logs. It is as per the value assigned while calling INFO, WARN, ERROR etc. logs.